### PR TITLE
introducing workpro puppet

### PR DESCRIPTION
**Pull request by hcfw007**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/jekyll/blob/main/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+categories: announcement
+tags:
+  - news
+  - puppet-service
+  - puppet-provider
+  - workpro
+image: /assets/2022/12-introducing-workpro-puppet/wecom.webp
+---
+
+经过长期的筹备与开发， WxWork 的继任者 workpro 作为 Wechaty 社区新的 Puppet Service 如今正式进入 beta 测试阶段了！欢迎社区的小伙伴申请7天的免费测试 token ，正式的运营方案也将很快出台，敬请期待！
```
Comment (windmemory) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:13]: WorkPro

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Wechaty Puppet Service Workpro 发布公告"
```
Comment (windmemory) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:2]: WorkPro

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  - puppet-provider
+  - workpro
+image: /assets/2022/12-introducing-workpro-puppet/wecom.webp
+---
+
+经过长期的筹备与开发， WxWork 的继任者 workpro 作为 Wechaty 社区新的 Puppet Service 如今正式进入 beta 测试阶段了！欢迎社区的小伙伴申请7天的免费测试 token ，正式的运营方案也将很快出台，敬请期待！
+
+## 关于 Wechaty Puppet Service WorkPro
+
+WorkPro 是 [juzibot](https://juzibot.com/) 基于 wechaty-puppet-service 做的新 puppet service，句子互动围绕微信生态为客户提供智能营销和销售服务，帮助企业引流并实现转化，客户覆盖教育、保险、大健康等多个领域。曾入选百度AI加速器 和 Facebook 大陆首期加速器，获得PreAngel、Plug and Play，Y Combination, TSVC和阿尔法公社多家中美机构投资。
```
Comment (lijiarui) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:17]: Change to RPAChat: http://rpachat.com/

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- CPU 和内存占用巨大
+- 当收到企微官方群发时，系统需要花费数小时来处理消息，导致无法使用
+- 当扫码登录时，会花很长时间来同步历史消息
+
+WorkPro 利用全新设计的架构彻底解决了以上问题，带来了效率与性能的巨大提升。他还包含了许多新功能，其中一些在社区版的 wechaty 上还无法支持。我们正在积极的推动这些功能合并入社区分支。
+
+## 关于 wechaty 的版本
+
+WorkPro 由于支持了群公告事件等新事件，目前与社区版的 wechaty 兼容存在一些问题，这会很快修复。详情请参考 [issue 216](https://github.com/wechaty/puppet-service/issues/216) 。
+WorkPro 的架构是直接根据 wechaty-grpc 的相关功能，对齐的是 wechaty-grpc 的最新版，对于 deprecated 的功能未做全面兼容，因此对于 wechaty 0.x 的兼容性无法保证。 wechaty 1.x 已经发布相当长的时间，建议使用最新版本的 wechaty。
```
Comment (windmemory) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:30]: Wechaty

**Review by windmemory — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md (hunk 3 above)
Comment (comeonzhj) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:17]: [RPAChat](http://rpachat.com/) 来自美国硅谷，专注于通过 RPA 技术将 IM 软件变成机器人。RPAChat 正在为宝洁、欧莱雅、Swisse、泡泡玛特等知名品牌提供基于对话的营销云解决方案。

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## 如何申请试用
+
+对于社区感兴趣的朋友，我们提供7天的免费试用 token ，请用微信或企微扫码添加客服咨询。
```
Comment (comeonzhj) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:34]: 这里是“对社区感兴趣”么？还是“对 puppet 感兴趣”？

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+![qrcode.webp](/assets/2022/12-introducing-workpro-puppet/qrcode.webp)
+
+## WorkPro Getting-Started
+
+对于想试用的朋友，可以参考[这个仓库](https://github.com/juzibot/workpro-getting-started)，也可以直接点击进入[社区版](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=581390026)或者[体验版](https://github.com/codespaces/new?hide_repo_select=true&ref=latest-features&repo=581390026)的 CodeSpace ，直接填入 token 即可开始使用。
```
Comment (comeonzhj) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:39]: 这个仓库是不是要放在 RPAChat https://github.com/rpachat 下面

**Review by comeonzhj — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2022-12-23-introducing-workpro-puppet.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+      token: 'xxx', // 将你的 token 填写在此，它应该形式类似于 'puppet_workpro_xxxxxx'
+      tls: {
+        disable: true, // 目前我们尚不支持 TLS 链接。详情请参考 : https://github.com/wechaty/puppet-service/issues/160
+      }
+    }
+    ```
+
+4. 使用 ```npm start``` 启动 bot ，你应该能看见二维码出现在你的终端中。
+
+## WorkPro 与 WxWork 的对比
```
Comment (lijiarui) [inline jekyll/_posts/2022-12-23-introducing-workpro-puppet.md:69]: 下面如果是图表，会更清晰直观

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Good job！ Cheers for RPAChat new puppet workpro！

**Pull request merged**